pkg/client/apiutil/restmapper.go

```diff
@@ -246,10 +246,18 @@ func (m *mapper) addGroupVersionResourcesToCacheAndReloadLocked(gvr map[schema.G
 		}
 
 		if !found {
-			groupResources.Group.Versions = append(groupResources.Group.Versions, metav1.GroupVersionForDiscovery{
+			doc := metav1.GroupVersionForDiscovery{
```

Comment:
What is `doc` a short-name for?

Reply:
document

Reply:
```suggestion
			gv := metav1.GroupVersionForDiscovery{
```

Short for GroupVersion. Just a suggestion. A short variable name makes sense here since the scope of the variable is short. Curious to know what doc/document has to do with this. 🤔